### ISSUE TITLE
multiple or adjustable whiteboard 

### ISSUE THREAD

**Issue by FlorianJa** [FlorianJa/BeamLite-Unity#7]
Integrate a function where the user can create another whiteboard. Also give the possibility to move the whiteboard.

**Comment by Grosskopf** [comment]
Looks related to / duplicate of #8 am i right? or should the post-it be locked from the other Players?

**Comment by FlorianJa** [comment]
Yes, basically a post it should be just a small whiteboard but with a different color (maybe yellow).
